popup.html

```diff
@@ -1,72 +1,226 @@
-<!doctype html>
```

Comment:
You have changed the syntax of the whole HTML file, for example:
![image](https://user-images.githubusercontent.com/31940194/97100190-2a2d8180-1667-11eb-9ec5-58a8981b91c1.png)
I haven't seen this practice before so I'd like to have it reverted back please, let's keep it the way it was!